packages/cinterop/src/jvmMain/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.4.1)
+project(RealmKotlin)
+find_package(JNI)
+set(SWIG_JNI "${CMAKE_SOURCE_DIR}/../../../jni-swig-stub/src/main/jni")
+
+set(CMAKE_CXX_STANDARD 14)
+add_subdirectory("${CMAKE_SOURCE_DIR}/../../../external/core" core EXCLUDE_FROM_ALL)
+
+include_directories("${CMAKE_SOURCE_DIR}/core/src" "${JAVA_INCLUDE_PATH}" "${JAVA_INCLUDE_PATH}/darwin")
```

Comment:
Why having a Darwin specific path here? Just a hardcoded path as we only have build in place for Darwin?

Reply:
As stated this PR covers only Darwin, to build Win/Linux we're going to have either a dedicated CMakeLists or Docker process that we spawn from the c-interop Gradle task